extensions/wikia/Helios/User.class.php

```diff
@@ -344,6 +344,17 @@ public static function onUserCheckPassword( $id, $username, $hash, $password, &$
 		return true;
 	}
 
+	/*
+	 * Invalidate all tokens for given user in helios service
+	 */
+	public static function onUserSetPassword( $userId, $forceLogout ) {
```

Comment:
I would name those two parameters the same in `OnUserSetPassword` and `setPassword`